webpack/components/extensions/HostDetails/Cards/ErrataOverviewCard.js

```diff
@@ -33,74 +38,113 @@ function HostInstallableErrata({
+                    left: 20,
+                    right: 140,
+                    top: 20,
+                  }}
+                  width={250}
+                  height={130}
                 />
+              </div>
+            </div>
+            <div className="erratalegend" style={{ minWidth: '140px' }}>
+              <FlexItem>
+                <ErrataMapper data={chartData} id={id} errataCategory={errataCategory} />
               </FlexItem>
-              <Flex flexWrap={{ xl: 'nowrap' }} direction="row" alignItems={{ default: 'alignItemsCenter' }}>
-                <div className="piechart-overflow" style={{ overflow: 'visible', minWidth: '140px', maxHeight: '155px' }}>
-                  <div className="erratachart" style={{ minWidth: '300px', minHeight: '300px' }}>
-                    <ChartPie
-                      ariaDesc="errataChart"
-                      data={chartData}
-                      constrainToVisibleArea
-                      labels={({ datum }) => `${datum.y} ${datum.w}`}
-                      padding={{
-                        bottom: 20,
-                        left: 20,
-                        right: 140,
-                        top: 20,
-                      }}
-                      width={250}
-                      height={130}
-                    />
-                  </div>
-                </div>
-                <div className="erratalegend" style={{ minWidth: '140px' }}>
-                  <FlexItem>
-                    <ErrataMapper data={chartData} id={id} />
-                  </FlexItem>
-                </div>
-              </Flex>
-            </Flex>
-          }
-        </CardBody>
-      </Card>
-    </GridItem>
+            </div>
+          </Flex>
+        </Flex>
+      }
+    </CardBody>
   );
 }
 
 const ErrataOverviewCard = ({ hostDetails }) => {
-  if (hostIsRegistered({ hostDetails }) && hostDetails.content_facet_attributes) {
-    const { id: hostId, errata_status: errataStatus } = hostDetails;
-    return (<HostInstallableErrata
-      {...propsToCamelCase(hostDetails.content_facet_attributes)}
-      id={hostId}
-      errataStatus={errataStatus}
-    />);
-  }
-  return null;
+  const hostPopulated = (hostIsRegistered({ hostDetails }) &&
+    !!hostDetails.content_facet_attributes);
+
+  const [errataCategory, setErrataCategory] = useState('applicable');
+  if (!hostPopulated) return null;
+  const {
+    id: hostId,
+    errata_status: errataStatus,
+    errata_status_label: errataStatusLabel,
+  } = hostDetails;
+  const { neededErrata } = errataStatusContemplation(errataStatus);
+  return (
+    <GridItem rowSpan={1} md={6} lg={4} xl2={3} >
+      <Card ouiaId="errata-card">
+        <CardHeader>
+          <Flex spaceItems={{ default: 'spaceItemsXl' }}>
+            <CardTitle>{__('Errata')}</CardTitle>
+            {neededErrata &&
+              <ToggleGroup isCompact>
+                <ErrataToggleGroupItem
+                  text={__('Applicable')}
```

Comment:
Should we add `aria-label` to these or just stick with text? Same for below.

Reply:
good point! Updated.